### ISSUE TITLE
Change logic of autorandr-lid-listener.service to make it more elegant

### ISSUE THREAD

**Issue by petRUShka** [phillipberndt/autorandr#333]
Right now it is just executing libinput debug-events every 30 seconds and looks at the output for a lid event. That seems like a dirty hack. 

There should be some solution to hook into acpid or something like that. Something like this [approach](https://mensfeld.pl/2018/08/ubuntu-18-04-disable-screen-on-lid-close/) or [similar](https://askubuntu.com/a/526003).

**Comment by laur89** [comment]
> it is just [executing](https://github.com/phillipberndt/autorandr/blob/master/contrib/systemd/autorandr-lid-listener.service) libinput debug-events every 30 seconds

Nope. The command specified starts a listener that runs indefinitely, assuming it doesn't crash. Only when the listener loop exits, does the [RestartSec](https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html#RestartSec=) come to play.

However I'm also curious as to why a systemd service was chosen for this function as opposed to a script in `/etc/acpi/events/` with contents to the tune of
```sh
event=button/lid
action=autorandr --batch --change --default default
```
